LOC.json

```diff
@@ -1865,9 +1865,9 @@
    "associatedRunwayNumber": "21",
    "frequency": 110300,
    "elevation": 20,
-   "latitude": 5.2737943,
-   "longitude": -3.92069811,
-   "bearing": 202.32,
+   "latitude": 5.27378051,
+   "longitude": -3.92070316,
+   "bearing": 22.34,
```

Comment:
Same as in glideslope either the heading or the runway number is wrong

Reply:
Bearing fixed; my mistake on that one.